### PR TITLE
[AL-0] Clean up Foundry docstrings

### PR DESCRIPTION
**Pull request by kkim-labelbox**
Cleaned up some docstrings around Foundry and Send to Annotate

### CODE REVIEW

**Review by karenkyang — APPROVED** [review]

File: labelbox/schema/send_to_annotate_params.py [hunk 1]
```diff
@@ -12,21 +12,24 @@
...
-        ids. Defaults to an empty dictionary.
+    Extra parameters for sending data rows and their predictions and/or annotations from Catalog to a specified project. 
+    
+    Note: 
+        At least one of source_model_run_id or source_project_id must be provided.
+
+    :param source_model_run_id: Optional[str] - The model run to use for sending predictions to a project. Defaults to None.
+    :param predictions_ontology_mapping: Optional[Dict[str, str]] - A dict of feature schema ids to feature schema
+        ids to map feature schemas from source model run's ontology to target project's ontology.
+        Defaults to an empty dictionary.
```
Comment (whistler) [inline labelbox/schema/send_to_annotate_params.py:23]: Shouldn't this be required if a source_model_run_id is provided? Is the empty default when a source_model_run_id is not specified? Maybe we should clarify this.

Same comment for `annotations_ontology_mapping`

File: labelbox/schema/model_run.py [hunk 2]
```diff
@@ -574,21 +574,28 @@ def send_to_annotate_from_model(
...
             >>> task = model_run.send_to_annotate_from_model(
             >>>     destination_project_id=DESTINATION_PROJECT_ID,
             >>>     batch_name="batch",
-            >>>     data_rows=UniqueIds([DATA_ROW_ID]),
+            >>>     data_rows=DataRowIds([DATA_ROW_ID]), # or GlobalKeys([GLOBAL_KEY])
             >>>     task_queue_id=TASK_QUEUE_ID,
-            >>>     params={})
+            >>>     params={
+            >>>         "predictions_ontology_mapping": {
+            >>>             "feature_schema_id": "feature_schema_id"
```
Comment (kozikkamil) [inline labelbox/schema/model_run.py:587]: nit: would be best to give customers clear understanding of what these feature schema ids represent.

`"model_run_schema_id": "destination_project_schema_id"`

File: labelbox/schema/send_to_annotate_params.py (hunk 1 above)
Comment (kozikkamil) [reply labelbox/schema/send_to_annotate_params.py:23]: Yeah it's required

File: labelbox/schema/send_to_annotate_params.py [hunk 3]
```diff
@@ -41,15 +44,19 @@ class SendToAnnotateFromCatalogParams(TypedDict):
...
+    :param predictions_ontology_mapping: Dict[str, str] - A dict of feature schema ids to feature schema
+        ids to map feature schemas from source model run's ontology to target project's ontology.
         Defaults to an empty dictionary.
     :param exclude_data_rows_in_project: Optional[bool] - Exclude data rows that are already in the project. Defaults
         to False.
-    :param override_existing_annotations_rule: Optional[ConflictResolutionStrategy] - The strategy defining how to
-        handle conflicts in classifications between the data rows that already exist in the project and incoming
-        predictions from the source model run. Defaults to ConflictResolutionStrategy.KEEP_EXISTING.
+    :param override_existing_annotations_rule: Optional[ConflictResolutionStrategy] - Strategy for resolving conflicts 
+        between existing classifications in the project and incoming predictions.
```
Comment (kozikkamil) [inline labelbox/schema/send_to_annotate_params.py:58]: nit: incoming prediction classifications?

**Review by tomislav-peharda — APPROVED** [review]

**Comment by adrian-chang** [comment]
@kkim-labelbox this is being closed until you are ready to merge / fix up

**Pull request closed without merging**